### DISCUSSION TITLE
How to setup a joined inheritance mapping with differents columns name ?

### DISCUSSION THREAD

**Discussion by nthnptt** [doctrine/doctrine2#8788] [Support Questions]
Hi !

I'm using doctrine for getting data from an external data source. I need to create an joined inheritance type between two tables. But tables has linked by columns with different name. There is a way to setup a joined inheritance mapping with different columns name ?

Thanks for your help.

**Comment by beberlei** [comment]
@yaman0 Why do you "need" tomake a joined inheritance type? Couldn't you also create an aggregation with a OneToOne or ManyToOne association? That would be more flexible w.r.t. to the column names as you can use the `@JoinColumn` to specify the names explicitly.

**Reply by nthnptt** [reply]
Sorry I miss to explain that the 2nd table depend of the 'type' of the first table.
Example : 
![uml (1)](https://user-images.githubusercontent.com/36821258/123555440-025a6480-d786-11eb-8972-28ce5b2f8f70.png)
